stable/weave-cloud/templates/cortex.yaml

```diff
@@ -130,7 +130,7 @@ items:
           serviceAccountName: weave-cortex
           tolerations:
             - effect: NoSchedule
-              key: node-role.kubernetes.io/master
+              operator: Exists
```

Comment:
No. Unfortunatelly [the docs](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#taints-and-tolerations-beta-feature) don't cover this, but it's clear from [code comments](https://github.com/kubernetes/kubernetes/blob/d2af14110528221ee532ba394a3bf66fe994a209/staging/src/k8s.io/api/core/v1/toleration.go#L29-L36):

```Go
// The matching follows the rules below:
// (1) Empty toleration.effect means to match all taint effects,
//     otherwise taint effect must equal to toleration.effect.
// (2) If toleration.operator is 'Exists', it means to match all taint values.
// (3) Empty toleration.key means to match all taint keys.
//     If toleration.key is empty, toleration.operator must be 'Exists';
//     this combination means to match all taint values and all taint keys.
```